spel/scripts/amigen8-build.sh

```diff
@@ -25,6 +25,9 @@ CLOUDPROVIDER="${SPEL_CLOUDPROVIDER:-aws}"
 DEBUG="${DEBUG:-UNDEF}"
 ELBUILD="/tmp/el-build"
 
+read -r -a BUILDDEPS <<< "${SPEL_BUILDDEPS:-lvm2 parted yum-utils unzip git}"
```

Comment:
This was three packages, going to five? Just checking that was deliberate, or copy pasta?
```suggestion
read -r -a BUILDDEPS <<< "${SPEL_BUILDDEPS:-lvm2 unzip git}"
```

Reply:
I'll have to go back and doublecheck `parted` and `yum-utils`: I'd _thought_ they'd become part of `@Core`, but it may have just been that they were baked in to the public RHEL 8.2 and MarketPlace CentOS AMIs I was starting with.

Was _gonna_ quick-check my local VMs …but it would seem that Hyper-V's networking is being a butt, again.

Reply:
Alright, so, yeah, **`parted`** is now part of `@Core`:

~~~
# yum groupinfo -v core | sed -n "1,/ Default Packages/p" | grep parted
   parted-3.2-38.el8.x86_64                                            @System
~~~

So, that one doesn't need to be in that list, but I'll need to add `yum-utils` back.